server/index.js

```diff
@@ -3,21 +3,37 @@ const path = require('path');
 const cookieParser = require('cookie-parser');
 const api = require('./routers/api');
 const main = require('./routers/main');
+const auth = require('./routers/auth');
+const admin = require('./routers/admin');
 const createApp = require('./app');
 require('dotenv').config({ path: path.join(__dirname, '../.env') });
 
 const port = process.env.PORT || 8080;
 const dbConn = process.env.DB_CONN;
 const cookieKey = process.env.COOKIE_KEY;
+const bodyCheckMiddleware = (request, response, next) => {
+  if (!request.body) {
+    return response.sendStatus(400);
+  }
+  return next();
+};
 
 createApp(dbConn, cookieKey).then((app) => {
   app.use(express.json());
   app.use(cookieParser(cookieKey));
 
   // routing
   app.use('/api', api);
+  app.use('/auth', auth);
+  app.use('/admin', admin);
   app.use('/', main);
 
+  // middleware
+  app.post('*', bodyCheckMiddleware);
+  app.put('*', bodyCheckMiddleware);
+  app.patch('*', bodyCheckMiddleware);
+  app.delete('*', bodyCheckMiddleware);
+
   // give static
   app.use(express.static(path.join(__dirname, '../dist')));
```

Comment:
@IgorChaikin If I logged in, and navigating to admin dashboard, after page refresh I can see only `json` data, but not admin dashboard.

![routing](https://user-images.githubusercontent.com/12561302/132857410-1346802d-5446-48e9-8535-0bc00d215b8b.gif)

You can add `/api/` prefix to all api routes, and use wildcard pattern `*` to send static filed to all other routes